addon/webextension/background/main.js

```diff
@@ -14,6 +14,9 @@ window.main = (function () {
 
   browser.storage.local.get(["hasSeenOnboarding"]).then((result) => {
     hasSeenOnboarding = !! result.hasSeenOnboarding;
+    if (! hasSeenOnboarding) {
+      setIconActive(false, null);
```

Comment:
nit: maybe we could leave the `null` off of these calls and default `tabId` to null. later is fine